src/usb_moded-config.c

```diff
@@ -334,12 +335,26 @@ int set_config_setting(const char *entry, const char *key, const char *value)
 {
   GKeyFile *settingsfile;
   gboolean test = FALSE;
-  int ret = 0; 
+  int ret = SET_CONFIG_ERROR;
   gchar *keyfile;
 
   settingsfile = g_key_file_new();
   test = g_key_file_load_from_file(settingsfile, FS_MOUNT_CONFIG_FILE, G_KEY_FILE_NONE, NULL);
-  if(!test)
```

Comment:
Might be worth moving that comparison logic to its own function. It might come in useful in other places, and avoids having this copy paste code in at least two places now... (function would be best defined in usb-moded-config.c). What do you think? I can of course do that myself quickly also.